libusb/usbdriver.h

```diff
@@ -178,14 +183,18 @@ typedef struct {
 } usb_pipeOps_t;
 
 
-struct usb_driver_t {
+struct _usb_driver_t {
+	struct _usb_transfer_t *next, *prev;
```

Comment:
This is supposed to be a circular doubly-linked list of `_usb_transfer_t` elements, right?